### DISCUSSION TITLE
Implementation of Distance Display in Kilometers for Users in Poland

### DISCUSSION THREAD

**Discussion by czuli** [code4recovery/12-step-meeting-list#1272] [Q&A] [answered]
### Contact Details

lukasz.czulak@gmail.com

### Requested Feature/Enhancement

Hi

I have noticed that in the current version of the "12-step-meeting-list" project, distances are displayed only in miles. I would like to request the implementation of an option to display distances in kilometers, which is the standard in Poland. This change would significantly facilitate the use of the application for users from my country, where the metric system is commonly used.

I suggest that users be given the option to select the unit of distance (miles or kilometers) in the application settings. This flexibility would not only improve the usability of the application in Poland but also in other countries where the metric system is used.

Thank you in advance for considering this proposal, and I look forward to its favorable consideration.

Sincerely,
cZuLi

**Comment by dougie-wougie** [comment, marked as answer]
I use miles, but I did see it in the setting. 
![CleanShot 2023-11-29 at 04 56 29](https://github.com/code4recovery/12-step-meeting-list/assets/58369176/e7671895-a2d6-4898-9a42-ce4aab1bfb5c)

**Reply by joshreisner** [reply]
@czuli let us know if this fixes it for you, so we can mark this as the answer, thanks!
